src/EditorFeatures/Core/Implementation/NavigationBar/NavigationBarController.cs

```diff
@@ -174,7 +165,7 @@ private void OnWorkspaceChanged(object sender, WorkspaceChangeEventArgs args)
                 var oldProject = args.OldSolution.GetProject(args.ProjectId);
                 var newProject = args.NewSolution.GetProject(args.ProjectId);
 
-                if (oldProject.Name != newProject.Name)
+                if (oldProject?.Name != newProject?.Name)
```

Comment:
Switch the two GetProject calls to GetRequiredProject, since the project would have to exist. (This hurts my head of what happens if the project doesn't exist on one side and whether the rest of the code is "correct" in such a state.)